src/utils/scripts.ts

```diff
@@ -77,7 +113,14 @@ export async function runScript(
     }
 
     for (const script of scripts) {
-        await execaCommand(script, { cwd, shell: true });
+        // await execaCommand(script, { cwd, shell: true });
+        await execSync(script, {
+            cwd,
+            env: {
+                ...process.env,
+                ...environment,
+            },
+        });
```

Comment:
why remove execa? they have a env option too: https://github.com/sindresorhus/execa/blob/main/docs/api.md#optionsenv that automatically extends the process.env

Reply:
I brought `execa` back :D